packages/manager/apps/pci-instances/src/components/statusChip/StatusChip.component.tsx

```diff
@@ -0,0 +1,23 @@
+import { ODS_THEME_COLOR_INTENT } from '@ovhcloud/ods-common-theming';
+import { OsdsChip } from '@ovhcloud/ods-components/react';
+import { useTranslation } from 'react-i18next';
+import { InstanceStatus } from '@/data/hooks/instances/useInstances';
+
+const StatusChip = ({ status }: { status: InstanceStatus }) => {
+  const { t } = useTranslation('status');
+
+  const colorBySeverityStatus = {
+    success: ODS_THEME_COLOR_INTENT.success,
+    error: ODS_THEME_COLOR_INTENT.error,
+    warning: ODS_THEME_COLOR_INTENT.warning,
+    info: ODS_THEME_COLOR_INTENT.info,
+  };
```

Comment:
this could be outside component to avoid recreating a new object each time

Reply:
Yep, good catch 👀